mailpoet/assets/js/src/automation/editor/components/header/more_menu.tsx

```diff
@@ -0,0 +1,35 @@
+import { MenuGroup } from '@wordpress/components';
+import { displayShortcut } from '@wordpress/keycodes';
+import { __, _x } from '@wordpress/i18n';
```

Comment:
just as a side note. not sure if we can with our current translation system utilize the javascript translation functions.

Reply:
Yeah... 🤔 This is pretty much copied from the post editor itself and I guess the strings that are identical may be available (although not sure if they need to be loaded) and it would be strange to rely on that anyway.

I have a "JS translations" point in the [checklist draft P2](https://wordpress.com/post/mailpoetdev.wordpress.com/1276) so maybe we could address that separately?

Reply:
Sure, just came to my mind when I saw it.